diagramming-tool/src/Component/HomePage/HomePage.js

```diff
@@ -0,0 +1,82 @@
+
+
+
+import React, { useState } from 'react';
+import './HomePage.css'; 
+import diagImage from './diag.png';
+
+import exampleImage from './example.jpg';
+import peopleImage from './people.png';
+import featuresImage from './features.png';
+import keImage from './ke.png'; 
+import dig from './dig.svg';
+
+const HomePage = () => {
+  const [showOptions, setShowOptions] = useState(false);
+
+  const toggleOptions = () => {
+    setShowOptions(!showOptions);
+  };
+
+  return (
+    <div className="home-page-container" data-testid="home-page-container"> {/* Add data-testid here */}
+      <img src={diagImage} alt="Diagram" className="diag-image" />
+      <header className="header">
+        <h1 className="title">Welcome to Our Diagramming Tool</h1>
+        <p className="subtitle">Visualize your ideas and concepts with ease</p>
+        {/* Move this inside */}
+        <nav className="nav"></nav>
```

Comment:
Usage of <nav> ? It is showing empty here.

Reply:
done changes